### ISSUE TITLE
[Northrend][Quest npc abilities] Smash - Back to the Pit

### ISSUE THREAD

**Issue by MrAndersonX** [dalaranwow/dalaran-wow#3930]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
 During the quest "Back to the Pit", the bear's Smash ability doesn't do the dmg indicated on the tooltip and it doesn't do the knock back effect.

**Expected behaviour**: Tell us what should happen instead
  The skill should deal 500 physical dmg (as told by the tooltip) and it should knock back the opponent allowing a charge after Smash.

**Steps to reproduce the problem**:

1.  Take the quest.
2.  Mount the bear.
3.  Use the ability.

**Include proofs for this behaviour**

https://postimg.org/gallery/j4xplemo/

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Rushor** [comment]
http://wotlk.openwow.com/quest=13424

**Comment by Rushor** [comment]
reason: the creature is imune to knackbacks cuz it's a vehicle, there must be added an excepion for it.

**Comment by Rushor** [comment]
Game/Vehicle: Enable knockback for Hyldsmeet Warbear and Lithe Stalker and add SPELL_ATTR0_CU_IGNORE_ARMOR for spell Smash

* closes https://github.com/dalaranwow/dalaran-wow/issues/3930
* commit: eca03c333d15db41da037d80876bc0c53158b4f4

![GitHub Logo](http://i.imgur.com/yp5ZZO6.jpg)

The issue was fixed by a developer and will now be tested on the Public Test Realm (PTR). The changes/fixes will be applied with the NEXT changelog on the liverealm.
This means that the issue exists in game at the moment i wrote this message.
Only once the issue was marked with "Fixed - On Live Server", you can find the fix on you server.
Please watch http://www.dalaran-wow.com/news/ to see when the changelog and the fix was applied.

**Comment by MrAndersonX** [comment]
@Rushor 
Gj man !

**Comment by TeridanDW** [comment]
Linked with #4309.

**Comment by Rushor** [comment]
This works for sure on my local ptr https://i.imgur.com/JOEnvKc.jpg, maybe the fix wasn't deployed to that time :)

**Comment by TeridanDW** [comment]
After a test on PTR this:

> Enable knockback for Hyldsmeet Warbear

doesn't seems to work properly as the bear casts the spell but I don't get knockback effect on me:
![wowscrnshot_111217_203436](https://user-images.githubusercontent.com/6797611/32702028-252a30c2-c7e9-11e7-856c-758e7e8a4916.jpg)

> SPELL_ATTR0_CU_IGNORE_ARMOR for spell Smash

Works properly on PTR.
